server/src/interfaces/Event.ts

```diff
@@ -0,0 +1,5 @@
+export interface Event {
+  timestamp: string;
+  id: string;
+  action: string;
```

Comment:
I'm assuming `action` will be its own type once we concretely identify what exact user behaviour we need to log? Like, I think in the end we won't want `action` to be able to be any arbitrary string?

Reply:
Yup! We can have some enum later on for all the valid actions.